crates/nu-protocol/src/eval_const.rs

```diff
@@ -259,39 +263,51 @@ pub(crate) fn create_nu_constant(engine_state: &EngineState, span: Span) -> Valu
     Value::record(record, span)
 }
 
-pub fn get_vendor_autoload_dir(engine_state: &EngineState) -> Option<PathBuf> {
+pub fn get_vendor_autoload_dir(_engine_state: &EngineState) -> Vec<PathBuf> {
```

Comment:
This is a little nitpick, but perhaps we should rename this to `get_vendor_autoload_dirs` as it now returns multiple values.

Reply:
done